### PR TITLE
Maid 1322

### PR DESCRIPTION
**Pull request by ustulation**

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @krishnaIndia

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: README.md [hunk 1]
```diff
@@ -39,5 +39,5 @@ cargo test
 ```
 ##TODO
 ### [0.0.1]
-- [ ] [MAID-1231](https://maidsafe.atlassian.net/browse/MAID-1321) safe_ffi - Make reading for file relative to serice home dir
-- [ ] [MAID-1232](https://maidsafe.atlassian.net/browse/MAID-1322) safe_ffi - changes according to other crates and update test cases
+- [X] [MAID-1231](https://maidsafe.atlassian.net/browse/MAID-1321) safe_ffi - Make reading for file relative to serice home dir
```
Comment (hitman401) [inline README.md]: The task Id is wrong in the TODO. Task Id is different in the link and in the text

File: src/implementation.rs [hunk 2]
```diff
@@ -33,30 +33,21 @@ pub fn path_tokeniser(c_path: *const ::libc::c_char) -> Result<Vec<String>, ::er
 
 pub fn get_final_subdirectory(client            : ::std::sync::Arc<::std::sync::Mutex<::safe_client::client::Client>>,
                               tokens            : &Vec<String>,
-                              starting_directory: Option<(&(::routing::NameType, u64), bool, bool)>) -> Result<::safe_nfs::directory_listing::DirectoryListing,
-                                                                                                         ::errors::FfiError> {
+                              starting_directory: Option<&::safe_nfs::metadata::directory_key::DirectoryKey>) -> Result<::safe_nfs::directory_listing::DirectoryListing,
+                                                                                                                        ::errors::FfiError> {
```
Comment (hitman401) [inline src/implementation.rs:37]: Empty line

Comment (ustulation) [inline src/implementation.rs:37]: there is no empty line

**Pull request merged**